src/services/blogService.js

```diff
@@ -0,0 +1,81 @@
+import database from '../database/models';
+class blogService {
+    static async getAllblogs() {
+        try {
+           const blog = await database.blog.findAll();
+           if(blog){
+               return blog;
+           }else{
+               return null;
+           }
```

Comment:
Unneeded, if `!blog` then it will return null
this should be the only statement in try `return await database.blog.findAll();`
This applies to multiple places